src/ipc/ipc-common/Registry.h

```diff
@@ -33,39 +33,61 @@
 #include <QMap>
 #include <functional>
 #include <QVector>
+#include <QDebug>
+
+struct RegistryEntry {
+    using ListenerFunction = std::function<void()>;
```

Comment:
do we really need this "using"? It makes things less readable. 
"Function" is enough as a name.

Reply:
Removed